modules/sagemaker/sagemaker-templates-service-catalog/templates/finetune_llm_evaluation/seed_code/build_app/source_scripts/requirements.txt

```diff
@@ -1,4 +1,4 @@
-transformers==4.38.0
+transformers[accelerate]==4.38.0
```

Comment:
To cross-link, I removed this in #206 as I was mistaken that it was unused. From conversations with @EthanBunce we realized that `transformers` uses it automatically if is installed. Adding it back as an extra will hopefully make it clear that we don't use it directly, but still want it installed.